Updated the daily quote section - added padding, colour and right-quote img added

vismitap opened the pull request:
Fixed #373

## Issue that this pull request solves

 Closes: #373 

## Proposed changes
The proposed changes were:
* The daily quote should appear on the development server
* The quote should have a good bottom padding
* The quote text color should be a bit nicer and attractive
### Brief description of what is fixed or changed
Changes made are:
* Added right quote image to give the quote a better and complete look. Earlier there was just on img `<img src="https://img.icons8.com/fluent-systems-filled/24/000000/quote-left.png"/>`, so closed it with `<img src="https://img.icons8.com/fluent-systems-filled/24/000000/quote-right.png"/>`.
* Changed the opacity and colors of both `quoteText` and `quoteAuthor` to make it look more attractive.
* Added padding at the bottom of quote section.
 
## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

Final output looks like this:
![image](https://user-images.githubusercontent.com/47056243/109066636-57706600-7713-11eb-957f-29acee56ee39.png)

goliakshay357 commented:
Hey @vismitap,
Thanks for your interest for solving this issue but I don't see the quotes when workspace are filled. 
**Screenshot**
![image](https://user-images.githubusercontent.com/32712438/109162139-e033e400-779d-11eb-933f-ef91779848b4.png)

Couple of suggestions: 
1. Following the color format would be great. (Same patterns of black is preferred)
2. Quote shall always be in the bottom of the page, even if the workspace is empty. 

Please feel free to message us on slack channel in case if you need any help or guidance.
Thanks

vismitap commented:
> Hey @vismitap,
> Thanks for your interest for solving this issue but I don't see the quotes when workspace are filled.
> **Screenshot**
> ![image](https://user-images.githubusercontent.com/32712438/109162139-e033e400-779d-11eb-933f-ef91779848b4.png)
> 
> Couple of suggestions:
> 
> 1. Following the color format would be great. (Same patterns of black is preferred)
> 2. Quote shall always be in the bottom of the page, even if the workspace is empty.
> 
> Please feel free to message us on slack channel in case if you need any help or guidance.
> Thanks

* I changed the colour because that was mentioned in the deliverable in #373, I can change that to black again if you want.
* I can see the quote even after adding the workspace.
![image](https://user-images.githubusercontent.com/47056243/109164414-69e4b100-77a0-11eb-8a32-2ff105dedf5d.png)
* The quote is always in the bottom as you can see in the screenshot. 
I don't know what the issue is in the deployment. 🤔

goliakshay357 commented:
Hey @vismitap,
Netlify provides us the flexibility of auto deployment preview of each PR. Can you message me on slack? 
Would love to debug together.

Thanks

vismitap commented:
> Hey @vismitap,
> Netlify provides us the flexibility of auto deployment preview of each PR. Can you message me on slack?
> Would love to debug together.
> 
> Thanks

Yes sure!

vismitap commented:
What's your username on slack? Could not find you on canvas board channel.

goliakshay357 commented:
> What's your username on slack? Could not find you on canvas board channel.
 
Display name: Goli Akshay Sujith
[Slack Invite](https://canvasbird-slack.netlify.app/)
Thanks

vismitap commented:
Hey! Added json file having a list of quotes in the `src/interfaces` folder and changed the color of quotes back to black. Added some padding to the container as well. I hope it works now! Here is the screenshot: 
![image](https://user-images.githubusercontent.com/47056243/109396001-62b2d400-7955-11eb-9ab8-038336834813.png)
